### PR TITLE
Fix broken links from DTA blog migration

### PR DESCRIPTION
**Pull request by sirleech**

### CODE REVIEW

**Comment by sirleech** [comment]
@zedzedzed @joolswood

File: content/digital-service-standard/11-measure-performance/guidance.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 layout: text/textblock
 ---
 
-[Measuring service performance](https://www.dta.gov.au/standard/measuring-performance/)
+[Measure success](/content-strategy/set-goals-measure-success/measure-success/)
```
Comment (joolswood) [inline content/digital-service-standard/11-measure-performance/guidance.md]: This page has a different focus from the original link. We'd need to find something more appropriate.

Comment (joolswood) [reply content/digital-service-standard/11-measure-performance/guidance.md]: I'm checking with the Insights team on this one.

File: content/digital-service-standard/11-measure-performance/reading.md [hunk 2]
```diff
@@ -5,4 +5,3 @@ layout: text/textblock
 - [GOV.UK Service Manual — Measuring success](https://www.gov.uk/service-manual/measuring-success)
 - [GOV.UK performance dashboards](https://www.gov.uk/performance)
 - [analytics.usa.gov](https://analytics.usa.gov)
-- [Digital Transformation Agency Blog — Performance Dashboard: practising what we preach](https://www.dta.gov.au/blog/dashboard-practising-what-we-preach/)
```
Comment (joolswood) [inline content/digital-service-standard/11-measure-performance/reading.md]: Checking with Insight team on this one as well

File: content/digital-service-standard/8-make-source-code-open/reading.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ layout: text/textblock
 - [GitHub and Government](https://government.github.com)
 - [Government GitHub Community](https://government.github.com/community/)
 - [Wikipedia — Comparison of free and open-source software licences](https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses)
-- [Digital Transformation Agency Blog — Making source code open](https://www.dta.gov.au/blog/making-source-code-open/)
+- [Digital Transformation Agency Blog — Making source code open](https://www.dta.gov.au/blogs/making-source-code-open)
```
Comment (joolswood) [inline content/digital-service-standard/8-make-source-code-open/reading.md]: ✅

File: content/topics/service-design-delivery-process/whole-user-experience/intro.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ title: Own the whole user experience
 subtitle: A service may have multiple products built by different teams — you need a service manager who understands and owns the whole user experience.
 ---
 
-We know that [people struggle to get things done with government](https://www.dta.gov.au/blog/gov-au-is-a-mental-model-for-government/) because services are fragmented. We force people to learn how government is structured to do what they need to do.
+We know that people struggle to get things done with government because services are fragmented. We force people to learn how government is structured to do what they need to do.
```
Comment (joolswood) [inline content/topics/service-design-delivery-process/whole-user-experience/intro.md]: ✅

File: content/topics/service-design-delivery-process/whole-user-experience/needs.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: text/textblock
 ---
-To build services that are complete and reflect the whole user experience we need to [start with needs: user needs, not government needs](https://www.dta.gov.au/standard/design-principles/#start-with-needs-user-needs-not-government-needs).
+To build services that are complete and reflect the whole user experience we need to start with needs: user needs, not government needs.
```
Comment (joolswood) [inline content/topics/service-design-delivery-process/whole-user-experience/needs.md]: ✅

File: content/digital-service-standard/11-measure-performance/reading.md (hunk 2 above)
Comment (joolswood) [reply content/digital-service-standard/11-measure-performance/reading.md]: ✅

File: content/digital-service-standard/11-measure-performance/guidance.md (hunk 1 above)
Comment (joolswood) [reply content/digital-service-standard/11-measure-performance/guidance.md]: @sirleech thanks ok after talking to the Insights team can we delete this partial and the `Guidance related to this criteria` heading partial (as there aren't any other guidance links) Happy for you to do this or us to pull this PR and update on our end.

File: content/digital-service-standard/11-measure-performance/index.yml [hunk 6]
```diff
@@ -14,10 +14,6 @@ main:
   - how.md
   - guidance-head.md
   - guidance.md
-  - reading-head.md
```
Comment (joolswood) [inline content/digital-service-standard/11-measure-performance/index.yml]: How come we're deleting `reading-head.md` and `reading.md` lines as well @zedzedzed?

Comment (zedzedzed) [reply content/digital-service-standard/11-measure-performance/index.yml]: I thought that's what you wanted in https://github.com/govau/service-manual/pull/771#discussion_r242402956 ? Ie delete the partial and the heading.

Comment (joolswood) [reply content/digital-service-standard/11-measure-performance/index.yml]: Sorry I wan't clearer. Only the `guidance.md` and `guidance-head.md` partials and the references to those partials in the` index.yml`.

Comment (zedzedzed) [reply content/digital-service-standard/11-measure-performance/index.yml]: Oh good pickup! I removed the wrong ones. Should be back now.

**Pull request merged**